localstack-core/localstack/aws/data/sqs-query/2012-11-05/service-2.json

```diff
@@ -1166,6 +1166,9 @@
         "AttributeNames":{
           "shape":"AttributeNameList"
         },
+        "MessageSystemAttributeNames":{
+          "shape":"AttributeNameList"
+        },
```

Comment:
I am afraid we might loose the overview of how we modified the original query spec. There would still be our version control, but in fact we already have a very well working way of maintaining these patches:
https://github.com/localstack/localstack/blob/930aac67c6516e58428b59dd3520c9d79ae9f2f1/localstack-core/localstack/aws/spec-patches.json

This file contains JSON Patches which are applied to the services when being loaded in our spec loader:
https://github.com/localstack/localstack/blob/930aac67c6516e58428b59dd3520c9d79ae9f2f1/localstack-core/localstack/aws/spec.py#L64-L70

Your changes here would be equivalent to the following addition to the `spec-patches.json`:
```json
  "sqs-query/2012-11-05/service-2": [
    {
      "op": "add",
      "path": "/shapes/ReceiveMessageRequest/members/MessageSystemAttributeNames",
      "value": {
        "shape": "AttributeNameList"
      }
    }
  ],
```

For the sake of transparency / keeping track of the changes, I would prefer this way.
Another option would be to add a note to the list of modifications on the internalized specs:
https://github.com/localstack/localstack/blob/930aac67c6516e58428b59dd3520c9d79ae9f2f1/localstack-core/localstack/aws/data/sqs-query/2012-11-05/README.md?plain=1#L7-L8